rmgpy/data/thermo.py

```diff
@@ -1356,54 +1364,66 @@ def get_thermo_data(self, species, training_set=None):
         # Return the resulting thermo parameters
         return thermo0
 
-    def set_delta_atomic_adsorption_energies(self, binding_energies=None):
+    def set_binding_energies(self, binding_energies=None):
         """
-        Sets and stores the change in atomic binding energy between
-        the desired and the Pt(111) default.
-
-        This depends on the two metal surfaces: the reference one used in
-        the database of adsorption energies, and the desired surface.
-
-        If binding_energies are not provided, resets the values to those
-        of the Pt(111) default.
+        Sets and stores the atomic binding energies specified in the input file.
 
         Args:
             binding_energies (dict, optional): the desired binding energies with
                 elements as keys and binding energy/unit tuples as values
 
         Returns:
-            None, stores result in self.delta_atomic_adsorption_energy
+            None, stores result in self.binding_energies
         """
-        reference_binding_energies = {
-            'C': rmgpy.quantity.Energy(-7.025, 'eV/molecule'),
-            'H': rmgpy.quantity.Energy(-2.754, 'eV/molecule'),
-            'O': rmgpy.quantity.Energy(-3.811, 'eV/molecule'),
-            'N': rmgpy.quantity.Energy(-4.632, 'eV/molecule'),
-        }
+        metal_db = MetalDatabase()
+        metal_db.load(os.path.join(settings['database.directory'], 'surface'))
 
-        # Use Pt(111) reference if no binding energies are provided
         if binding_energies is None:
-            binding_energies = reference_binding_energies
+            # Use Pt(111) reference that surfaceThermoPt111 was calculated on if no binding energies are provided
+            binding_energies = metal_db.get_binding_energies('Pt111')
+        elif binding_energies is str:
```

Comment:
Ummmm, I'll wager that's not what you meant.

(don't fix it now, I'm working on this bit of code)